point_viewer_grpc/src/service.rs

```diff
@@ -167,16 +95,23 @@ impl proto_grpc::Octree for OctreeService {
         req: proto::GetNodeDataRequest,
         sink: UnarySink<proto::GetNodeDataReply>,
     ) {
-        let data = self
+        let service_data = match self.get_service_data(&req.octree_id) {
+            Ok(service_data) => service_data,
+            Err(e) => return send_fail(&ctx, sink, &e),
+        };
+        let node_data = match service_data
             .octree
             .get_node_data(&NodeId::from_str(&req.id).unwrap())
```

Comment:
How about this `unwrap`? :D

Reply:
Ok, just handled this one, too. However, I would like to leave the remaining ones, since they are all pretty safe and replacing them would only clutter the source.